docs/getting-started/Typed_Linting.mdx

```diff
@@ -16,6 +16,8 @@ To tap into TypeScript's additional powers, there are two small changes you need
 2. Add `languageOptions.parserOptions` to tell our parser how to find the TSConfig for each source file.
 
 ```js title="eslint.config.mjs"
+import tseslint from 'typescript-eslint';
```

Comment:
Tangent: It's been bugging me that you can't copy & paste the full code snippet from this page. :smile: